jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/jdbc/JdbcAdapterTest.java

```diff
@@ -0,0 +1,94 @@
+package com.exasol.adapter.jdbc;
+
+import com.exasol.ExaConnectionAccessException;
+import com.exasol.ExaConnectionInformation;
+import com.exasol.ExaMetadata;
+import com.exasol.adapter.metadata.SchemaMetadata;
+import com.exasol.adapter.metadata.SchemaMetadataInfo;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.*;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class JdbcAdapterTest {
+    private ExaConnectionInformation exaConnectionInformation;
+    private ExaMetadata exaMetadata;
+
+    @Before
+    public void setUp() throws ExaConnectionAccessException {
+        exaConnectionInformation = mock(ExaConnectionInformation.class);
+        exaMetadata = mock(ExaMetadata.class);
+
+        when(exaConnectionInformation.getUser()).thenReturn("testCommUser");
+        when(exaConnectionInformation.getPassword()).thenReturn("testConnPassword");
+        when(exaConnectionInformation.getAddress()).thenReturn("testConnAddress");
+        when(exaMetadata.getConnection(any())).thenReturn(exaConnectionInformation);
+    }
+
+    @Test
+    public void getCredentialsForJDBCImportWithConnectionNameGiven() {
+        Map<String, String> properties = new HashMap<>();
```

Comment:
Add `final`.

Reply:
DRY: Turn properties into instance field and initialize in `setUp()`.

Reply:
done